src/content/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard.mdx

```diff
@@ -92,6 +92,13 @@ Dashboards features include:
     icon from the upper right menu bar.
   </Collapser>
 
+  <Collapser
+    id="dash-json"
+    title="Copy your dashboard as a JSON"
+  >
+    You can copy your dashboard as a JSON and add it to the clipboard by clicking on the **&lt;/&gt;** icon on the right corner.
```

Comment:
Hey @urbiz-nr, I wanted to point out an alternate syntax for those HTML entities that you have defined in this line. If you like the readability of using the `<` and `>` characters rather than using the html entities directly, you can also use a JSX expression for this. A JSX expression uses curly brackets and string syntax. It looks like this:

```mdx
You can copy your dashboard as a JSON and add it to the clipboard by clicking on the **{'</>'}** icon on the right corner.
```

By no means are you required to use this, just wanted to point this out in case you liked this better! Both methods work so pick your favorite 🙂

Reply:
Here, also, please take out the "a"  to make it "copy your dashboard as JSON" or "as JSON code"